client/src/app/shared/misc/utils.ts

```diff
@@ -4,6 +4,10 @@ import { DatePipe } from '@angular/common'
 import { environment } from '../../../environments/environment'
 import { AuthService } from '../../core/auth'
 
+type ElementEvent = Omit<Event, 'target'> & {
```

Comment:
I think you can remove this

Reply:
No: `timestamp-route-transformer.directive.ts` depends on it.